### PR TITLE
Support for arguments

### PR DESCRIPTION
**Pull request by asbjornenge**
I needed to pass a few flags to chrome. I have no means of testing outside Linux, so if it obviously breaks on other platforms, ignore it. :+1:

### CODE REVIEW

**Comment by sindresorhus** [comment]
Have you seen #6?

// @tunnckoCore

File: index.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ module.exports = function (target, app, cb) {
 
 	var cmd;
 	var args = [];
+    var app_args = [];
+    if (app instanceof Array) { app_args = app.splice(1); app = app[0] }
```
Comment (sindresorhus) [inline index.js]: code style please ;)

camelCase
semicolons
etc

File: test.js [hunk 2]
```diff
@@ -27,3 +27,11 @@ it('should open url in specified app', function (cb) {
 		cb();
 	});
 });
+
+it('should open url in specified app with arguments', function(cb) {
```
Comment (sindresorhus) [inline test.js]: `function (cb) {`

File: index.js (hunk 1 above)
Comment (sindresorhus) [inline index.js]:
```js
if (Array.isArray(app)) {
  appArgs = app.splice(1);
  app = app[0];
}
```

File: index.js [hunk 3]
```diff
@@ -46,6 +48,7 @@ module.exports = function (target, app, cb) {
 		}
 	}
 
+    args = args.concat(app_args);
```
Comment (sindresorhus) [inline index.js]: indent

**Comment by asbjornenge** [comment]
Can't you auto-format that stuff? Here is my styleguide https://github.com/asbjornenge/javascript-style :stuck_out_tongue: I saw #6 but felt this was a bit simpler. If you don't want the PR, just close it. I'll live with my fork, np :monkey:

**Comment by sindresorhus** [comment]
fixed in 2.0.0 => https://github.com/sindresorhus/opn/commit/55e6dac9264dc02844b64a870f1d6d40160452a9

**Pull request closed without merging**